src/util/dev/server.ts

```diff
@@ -552,7 +557,7 @@ export default class DevServer {
   validateNowConfig(config: NowConfig): void {
     if (config.version !== 2) {
       this.output.error('Only `version: 2` is supported by `now dev`');
-      process.exit(1);
+      this.exit();
```

Comment:
Are you missing `await` here?

Reply:
@styfle nope, since the function is not `async`, it'll take care of cleaning up and exiting, but it's not required to await.